README.md

```diff
@@ -1 +1,32 @@
-# notes-app
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.<br />
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+## Amplify commands for building and deploying
```

Comment:
This is fine for now but as you go along this should become specific instructions on how to build, dev on and deploy *this* project.

Reply:
That makes sense, thank you!